scripts/errors/fetch-errors.sh

```diff
@@ -0,0 +1,12 @@
+ERRORS_FOLDER=.errors
+
+echo "Fetching error specifications from backends"
+
+curl https://api.stage.openshift.com/api/kafkas_mgmt/v1/errors | jq > $ERRORS_FOLDER/errors_kafka_mgmt.json
+
+curl https://api.stage.openshift.com/api/serviceregistry_mgmt/v1/errors | jq > $ERRORS_FOLDER/errors_srs_mgmt.json
+
+## Not enabled yet
+# curl https://api.stage.openshift.com/api/connector_mgmt/v1/errors | jq > errors_connector_mgmt.json
+ 
+echo "Successfully fetched errors"
```

Comment:
A lot of repetition. If you include the base URL as part of the config metadata then it can figure out the rest (as the errors are standardized.
 
Even though it is a simple script, I do see it making more sense as part of the Go packages, for consistency more than anything.